tests/utils/iotrace.py

```diff
@@ -308,9 +304,9 @@ def get_fs_stats(trace_path: str, shortcut: bool = False) -> list:
         command += (' -p ' if shortcut else ' --path ') + f'{trace_path}'
         command += (' -f ' if shortcut else ' --format ') + 'json'
 
-        output = TestRun.executor.run(command).stdout
-        if output == "":
-            raise CmdException("Invalid statistics")
+        output = TestRun.executor.run(command)
+        if output.stdout == "":
+            raise CmdException("Invalid filesystem statistics", output)
 
         return parse_json(output.stdout)
```

Comment:
Before it was `return parse_json(output.stdout)[0]["entries"]`, and that's what the test expects.

Reply:
Ok, would check it and modify one of these.